include/miral/miral/window_manager_tools.h

```diff
@@ -161,6 +161,9 @@ class WindowManagerTools
     /// \remark Since MirAL 3.10
     auto window_to_select_application(const Application) const -> std::optional<Window>;
 
+    /// Check if the provided window can be selected
+    auto can_select_window(Window const&) const -> bool;
```

Comment:
```suggestion
    /// Check if the provided window can be selected
    /// \remark Since MirAL 5.0
    auto can_select_window(Window const&) const -> bool;
```